lib/dishes.rb

```diff
@@ -1,7 +1,21 @@
 class Dishes
   attr_accessor :menu
 
-  def view(menu)
-    @menu = menu
+  def initialize
+    @menu = {
```

Comment:
this is a good idea to have the menu at the initialize stage which is good use of low coupling.